### PR TITLE
doc: Improve the TRANSLATION.md

### PR DESCRIPTION
**Pull request by ghost**
The introduction of TRANSLATION.md for Crowdin is a bit too simple to understand, especially for a newbie who wants to join in the group and how to get started. Here I've given some very useful links to guide them step-by-step on how to cope with Crowdin and join in the translating group together to make contributions to nodejs.org.

Several changes:

1. Rename 'Crowdin' to 'Nodejs-website in Crowin', because this is a nodejs-website based Crowdin project.

2. Add more step-by-step guides to bring you to the home of Crowdin and join in it without problems to get started quickly.

3. For any discussions related to our previous discussion at https://github.com/nodejs/nodejs.org/discussions/5222#discussioncomment-5517664 and https://github.com/nodejs/nodejs.org/discussions/5222#discussioncomment-5518266, we'd better include such info to let volunteers get to know how to have discussions/chats in Crowdin instead of in GitHub.

### CODE REVIEW

**Comment by ghost** [comment]
I'm a newbie to bother others about Crowdin, and I don't like others bothering and questioning here and there, that's the reason why I want to improve this article, hope most frequently-asked questions can be solved here :)

**Review by AugustinMauroy — APPROVED** [review]

File: TRANSLATION.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-1. Request to join the [Crowdin](https://crowdin.com/project/nodejs-website)
-2. Select the language you want to translate
-3. Start translating
+1. Request to join the [Nodejs-Website in Crowdin](https://crowdin.com/project/nodejs-website)
+2. [Select the language you want to translate](https://support.crowdin.com/joining-translation-project/#starting-translation)
+3. [Start translating](https://support.crowdin.com/online-editor/)
+
+## Any questions or feedbacks on Translations
+
+If you have any questions or feedbacks on current translations, you can start a discussion [here](https://crowdin.com/project/nodejs-website/discussions) by clicking the "New Topic" and choosing your language from the right dropdown, or a [conversation](https://support.crowdin.com/enterprise/messages/#creating-a-new-conversation) by adding your translators.
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
If you have any questions or feedbacks on current translations, you can [start a discussion](https://crowdin.com/project/nodejs-website/discussions) by choosing "New Topic" and your language from the right dropdown, or a [conversation](https://support.crowdin.com/enterprise/messages/#creating-a-new-conversation) by adding your translators.
```

Comment (ghost) [reply TRANSLATION.md]: Thank you @Trott , haven't seen you for a long time

**Review by mikeesto — APPROVED** [review]

**Review by manishprivet — APPROVED** [review]

File: TRANSLATION.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
-1. Request to join the [Crowdin](https://crowdin.com/project/nodejs-website)
-2. Select the language you want to translate
-3. Start translating
+1. Request to join the [Nodejs-Website in Crowdin](https://crowdin.com/project/nodejs-website)
+2. [Select the language you want to translate](https://support.crowdin.com/joining-translation-project/#starting-translation)
+3. [Start translating](https://support.crowdin.com/online-editor/)
+
+## Any questions or feedbacks on Translations
+
+If you have any questions or feedbacks on current translations, you can [start a discussion](https://crowdin.com/project/nodejs-website/discussions) by choosing the "New Topic" and your language from the right dropdown, or a [conversation](https://support.crowdin.com/enterprise/messages/#creating-a-new-conversation) by adding your translators.
```
Comment (ovflowd) [inline TRANSLATION.md]:
```suggestion
If you have any questions or feedbacks on current translations, you can [start a discussion](https://crowdin.com/project/nodejs-website/discussions) by choosing the "New Topic" and your language from the right dropdown, or a [conversation](https://support.crowdin.com/conversations/) by adding your translators.
```

Comment (ghost) [reply TRANSLATION.md]: Many thanks! This is more suitable

**Review by ovflowd — APPROVED** [review]
LGTM! Thank you for updating the docs, @MaledongGit!

**Pull request merged**